IdentityCore/src/cache/MSIDSharedCacheAccessor.h

```diff
@@ -67,5 +74,14 @@
                         context:(id<MSIDRequestContext>)context
                           error:(NSError **)error;
 
+- (BOOL)removeAccount:(MSIDAccount *)account
+              context:(id<MSIDRequestContext>)context
+                error:(NSError **)error;
+
+- (BOOL)removeAllTokensForAccount:(MSIDAccount *)account
+                          context:(id<MSIDRequestContext>)context
+                            error:(NSError **)error;
+
+- (BOOL)clearWithContext:(id<MSIDRequestContext>)context error:(NSError **)error;
```

Comment:
that's a bit dangerous method, where do we use it?

Reply:
It is used in test app for cache clearing.

Reply:
done